src/components/Stack/tests/Stack.test.tsx

```diff
@@ -0,0 +1,254 @@
+import * as React from 'react';
+import {mount} from 'enzyme';
+import Stack from '../Stack';
+import Badge from '../';
+import Item from '../Item';
+import Heading from '../';
+
+describe('Stack Component Suit', () => {
```

Comment:
Change root level `describe('<Stack />', () => {` describe and have all tests inside it.